src/pms/__init__.py

```diff
@@ -1,3 +1,8 @@
+from loguru import logger
+
+logger.disable("pms")  # disable logging by default
```

Comment:
Not sure that disabling the longing is the right behaviour for library use.
In principle, the library user should decide the logging level.

Reply:
I have two problems with this:

1. The default logging level is "DEBUG", which is not a sensible default.

```
import loguru
loguru.logger.debug('test')
2023-01-07 10:27:45.491 | DEBUG    | __main__:<module>:1 - test
```

2. PyPMS logs now have to be configured using a different framework.

```
import logging
logging.configure({ ... })  # main logging for other libraries

import loguru
loguru.logger.configure(...)  # special config just for PyPMS
```

Normally I don't need DEBUG logs for PyPMS. If I do need them, then as a developer I am willing to go find out how to turn them on. Conversely, when I start using a library I don't want to spend extra effort working out how to switch all the DEBUG logs off to stop them polluting my normal / day-to-day logs.

`logger.disable("pms")` seemed like the easiest solution to get back to a sensible default for library users. I also think some documentation would be helpful, but that should be a secondary priority to just having some sensible defaults in the first place - the priority should be ease of use / minimal setup effort.